aws_adfs/login.py

```diff
@@ -231,12 +231,13 @@ def login(
         _emit_summary(config, aws_session_duration)
 
 
-def _emit_json(aws_session_token):
+def _emit_json(aws_session_token, aws_session_duration):
     click.echo(
-        u"""{{"AccessKeyId": "{}", "SecretAccessKey": "{}", "SessionToken": "{}"}}""".format(
+        u"""{{"AccessKeyId": "{}", "SecretAccessKey": "{}", "SessionToken": "{}", "Expiration": "{}", "Version": 1}}""".format(
             aws_session_token['Credentials']['AccessKeyId'],
             aws_session_token['Credentials']['SecretAccessKey'],
-            aws_session_token['Credentials']['SessionToken']
+            aws_session_token['Credentials']['SessionToken'],
+            (datetime.datetime.now()+datetime.timedelta(0,aws_session_duration)).isoformat()
```

Comment:
An import statement is missing for the `datetime` module at the top of the file.

Also, the generated date should always include a [timezone offset](https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat). I propose to use UTC in all cases:

```suggestion
            (datetime.datetime.now(UTC())+datetime.timedelta(0,aws_session_duration)).isoformat()
```

Failing to provide an offset-aware datetime triggers errors in the AWS CLI:

```
  File "/home/patrick/.local/pipx/venvs/awscli/lib/python3.8/site-packages/botocore/credentials.py", line 442, in _seconds_remaining
    delta = self._expiry_time - self._time_fetcher()
TypeError: can't subtract offset-naive and offset-aware datetimes
2020-06-30 14:51:35,910 - MainThread - awscli.clidriver - DEBUG - Exiting with rc 255

can't subtract offset-naive and offset-aware datetimes
```